src/Uno.UI/UI/Xaml/Controls/Image/Image.wasm.cs

```diff
@@ -201,7 +202,20 @@ private void SetImageUrl(string url)
 			}
 			else
 			{
-				_htmlImage.SetAttribute("src", url);
+				if (UNO_BOOTSTRAP_APP_BASE != null)
+				{
+					var adjustedUrl = url switch
+					{
+						string u when u.StartsWith("http:/") || u.StartsWith("https:/") => url,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this call to 'u.StartsWith' to an overload that accepts a 'StringComparison' as a parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5643263)